tests/mpi/distribute_mg_dofs_periodicity_01.mpirun=28.output

```diff
@@ -0,0 +1,7 @@
+
+DEAL::Number of cells: 1024
```

Comment:
In 3D, the problem only failed on 28 processors (the 6 processor case was for 2D). Thus, I think that it represents a case that previously failed but now runs. We could argue that the underlying problem appears in 2D already, so there is no real need to keep it.